src/ticket/lib/ticketReducer.ts

```
@@ -0,0 +1,78 @@
+export const ticketReducer = (
+  tickets: Ticket[],
+  action: TicketAction,
+): Ticket[] => {
+  switch (action.type) {
+    case 'ADD_TICKET':
+      const newTicket: Ticket = {
+        ...action.payload,
+        id: tickets.length + 1,
+        status: 'open',
+        comments: [],
+      };
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)